src/client/utils/hooks/useCreateQuoteCart.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback } from 'react'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useCreateOnboardingQuoteCartMutation } from 'data/graphql'
+import { useStorage } from './../StorageContainer'
+
+export const useCreateQuoteCart = () => {
```

Comment:
This is a great way to guarantee we do this all the time! Would have been great to be able to "ban" the use of the mutation directly

Reply:
Hmm yeah indeed, we could write our own eslint rule for that 😅